<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-996CEA1" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="2020-05-21T14:52:05 IST" time="0.112" errors="0">
  <testcase name="test2" time="0.018" classname="demo.TestNGRetryDemo"/>
  <testcase name="test3" time="0.006" classname="demo.TestNGRetryDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryDemo.test3(TestNGRetryDemo.java:24)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="demo.TestNGRetryDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
